src/Ledger/Introduction.lagda

```diff
@@ -199,32 +200,47 @@ we use here to form a type of sets with elements in a given type.
 
 \begin{figure*}[h]
 \begin{code}[hide]
-module _ (ℙ_ : Set → Set) (_∈_ : ∀ {A : Set} → A → ℙ A → Set) where
+module _ (ℙ_ : Type → Type) (_∈_ : ∀ {A : Type} → A → ℙ A → Type) where
   private variable
     a c : Level
-    A : Set a
-  Σ-syntax' : (A : Set a) → (A → Set c) → Set _
+    A : Type a
+  Σ-syntax' : (A : Type a) → (A → Type c) → Type _
   Σ-syntax' = Σ
   syntax Σ-syntax' A (λ x → B) = x ∈ A ﹐ B
 \end{code}
 \begin{code}
-  _⊆_ : {A : Set} → ℙ A → ℙ A → Set
+  _⊆_ : {A : Type} → ℙ A → ℙ A → Type
   X ⊆ Y = ∀ {x} → x ∈ X → x ∈ Y
 
-  _≡ᵉ_ : {A : Set} → ℙ A → ℙ A → Set
+  _≡ᵉ_ : {A : Type} → ℙ A → ℙ A → Type
   X ≡ᵉ Y = X ⊆ Y × Y ⊆ X
 
-  Rel : Set → Set → Set
+  Rel : Type → Type → Type
   Rel A B = ℙ (A × B)
 
-  left-unique : {A B : Set} → Rel A B → Set
+  left-unique : {A B : Type} → Rel A B → Type
   left-unique R = ∀ {a b b'} → (a , b) ∈ R → (a , b') ∈ R → b ≡ b'
 
-  _⇀_ : Set → Set → Set
+  _⇀_ : Type → Type → Type
   A ⇀ B = r ∈ Rel A B ﹐ left-unique r
 \end{code}
 \end{figure*}
 
+\subsection{Propositions as types, properties and relations}
+
+In type theory we represent propositions as types and proofs of a proposition as
+elements of the corresponding type.
+A \textit{predicate} \AgdaFunction{P} represents a property or a relation.
+A unary predicate is a function that takes each \AgdaBound{x} (of some type \AgdaBound{A}) and
+returns a proposition \AgdaFunction{P}(\AgdaBound{x}). Thus, a predicate is a function of type
+\AgdaBound{A}~\AgdaSymbol{→}~\Type.
+
+A \textit{binary relation} \AgdaFunction{R} from \AgdaBound{A} to \AgdaBound{B} is a
```

Comment:
```suggestion
A \textit{binary relation} \AgdaFunction{R} between \AgdaBound{A} and \AgdaBound{B} is a
```

Reply:
Often relations aren't directional

Reply:
You have a point.  I've been using that wording for years because I felt it would make it less jarring for students when I tell them a function from A to B is just a special kind of binary relation from A to B.  I would hate to think that actually *added* to their confusion. 😟  In any case, I'll go with your suggested change, which is probably more standard anyway.